server/routes/api/stats.py

```diff
@@ -20,12 +20,13 @@
 
 # Define blueprint
 bp = Blueprint(
```

Comment:
some how this one is not wrapped in one line..

Reply:
Oh, that's odd. Manually made it one line